src/grt/src/fastroute/src/maze.cpp

```diff
@@ -1005,7 +1006,15 @@ void FastRouteCore::updateRouteType1(const TreeNode* treenodes,
   }
 
   if (E1_pos == -1) {
-    logger_->error(GRT, 169, "Invalid index for position ({}, {}).", E1x, E1y);
+    int x_pos = w_tile_ * (E1x + 0.5) + x_corner_;
+    int y_pos = h_tile_ * (E1y + 0.5) + y_corner_;
+    logger_->warn(GRT,
+                   169,
+                   "Net {}: Invalid index for position ({}, {}). Net degree: {}.",
+                   netName(nets_[net_id]),
+                   x_pos,
+                   y_pos,
+                   nets_[net_id]->numPins);
```

Comment:
Is it safe to continue after this warning with E1_pos == -1?

Reply:
Yes, because the `checkRoute2DTree` function will detect the issue with the tree and will reset the tree.